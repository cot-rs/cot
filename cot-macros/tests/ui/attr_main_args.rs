#[cot::main]
async fn main(arg: i32) -> cot::Result<CotProject> {
    std::process::exit(0);
}
