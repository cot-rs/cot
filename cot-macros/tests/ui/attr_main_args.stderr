error: cot::main function must have zero arguments
 --> tests/ui/attr_main_args.rs:2:15
  |
2 | async fn main(arg: i32) -> cot::Result<CotProject> {
  |               ^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/attr_main_args.rs:4:2
  |
4 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/attr_main_args.rs`
